src/lib/config.ts

```diff
@@ -132,10 +131,20 @@ export const DB_QUERY_TIMEOUT = intVar(
 export const DELTA_HOST = requiredVar('DELTA_HOST');
 export const FILES_HOST = optionalVar('FILES_HOST', '');
 export const DEVICE_CONFIG_OPENVPN_CA = requiredVar('DEVICE_CONFIG_OPENVPN_CA');
+export const VPN_UDP_HOST = optionalVar('VPN_UDP_HOST');
+export const VPN_HOST = requiredVar('VPN_HOST');
+export const VPN_PORT = requiredVar('VPN_PORT');
+
+const remotes = [];
+if (VPN_UDP_HOST) {
+	remotes.push(`remote ${VPN_UDP_HOST} ${VPN_PORT} udp`);
+}
+remotes.push(`remote ${VPN_HOST} ${VPN_PORT} tcp`);
```

Comment:
I saw in the docs at some point that
> If you want OpenVPN to listen on both a UDP and TCP port, you must run two separate OpenVPN instances

See: https://openvpn.net/community-resources/how-to/#:~:text=If%20you%20want%20OpenVPN%20to%20listen%20on%20both%20a%20UDP%20and%20TCP%20port%2C%20you%20must%20run%20two%20separate%20OpenVPN%20instances

So should this be an `if-else`?